### ISSUE TITLE
error on cpp & neko target 

### ISSUE THREAD

**Issue by man2** [TiVo/intellij-haxe#302]
I always get error when running for cpp and neko target. flash target runs smoothly. I'm using haxeflixel hello world template  project. Is the plugin supposed to be working for other target besides flash?

**Comment by EBatTiVo** [comment]
Yes, Neko and cpp targets are supposed to work correctly.  What is the exact error you are seeing, and how do you reproduce it?  Haxeflixel is not something we test with.

**Comment by man2** [comment]
This is the error I'm getting :

Information:- Running command: java -Dapple.awt.UIElement=true -jar /usr/lib/haxe/lib/lime/2,4,4//templates/bin/batik/batik-rasterizer.jar -d /var/folders/r4/r4LB-k1jFlqvno4H63reT++++TI/-Tmp-//temp_7250885.png -w 16 -h 16 /usr/lib/haxe/lib/flixel/3,3,8/assets/images/logo/HaxeFlixel.svg
Error:
Information:
Information:[file_open,/var/folders/r4/r4LB-k1jFlqvno4H63reT++++TI/-Tmp-//temp_7250885.png]
Information:Error while converting files:
Error:unable to create output directory

I just created flixel template for Intellij Idea editor with
flixel tpl -n "hello world"

then open the generated iml file with Intellij Idea.

Choose neko target in Project structure | Modules | helloworld | Haxe | Target

Boom,  I got 2 errors, first error : no error message
second error : Error while converting files:
Error:unable to create output directory

Any ideas?

**Comment by man2** [comment]
I also encounter this error on standard openfl project

Error:
Information:
Information:[file_open,/var/folders/r4/r4LB-k1jFlqvno4H63reT++++TI/-Tmp-//temp_8090335.png]
Information:- Running command: java -Dapple.awt.UIElement=true -jar /usr/lib/haxe/lib/lime/2,4,4//templates/bin/batik/batik-rasterizer.jar -d /var/folders/r4/r4LB-k1jFlqvno4H63reT++++TI/-Tmp-//temp_8090335.png -w 16 -h 16 Assets/openfl.svg
Information:About to transcode 1 SVG file(s)
Information:Error while converting files:
Error:unable to create output directory

This only happen for neko & cpp target. Flash is working just fine

**Comment by EBatTiVo** [comment]
My first observation is that the errors are coming from batik-rasterizer, not IDEA or the Haxe plugin.  Second, the message is stating that the output directory cannot be created.  The questions are: Does the process have permissions to create the output directory; and does this command work from the command line, outside of IDEA?  

If the same command works from the command line outside of IDEA, then the issue is likely in the project configuration.  If it does not, then the issue is the environment -- and then there may still be an issue with the project configuration.  Either way, it doesn't look much like a plugin bug.

That said, the output file names are somewhat odd temporary names and may be being generated incorrectly for your OS.  What OS are you running this on?

**Comment by EBatTiVo** [comment]
On my Linux box, the /var/. folder permissions are owned by root, and I cannot `mkdir -p /var/folders/r4/r4LB-k1jFlqvno4H63reT++++TI/-Tmp-//temp_8090335.png`, so I would get the same result as you.

It would be interesting to see what command line is used in the successful case.

I do not see any code in the intellij plugin or intellij-community sources (14.1.3) that specifies /var/folders.

**Comment by man2** [comment]
I'm using mac osx 10.10.3
